open_group.py

```diff
@@ -11,24 +9,19 @@ class Facebook:
     def __init__(self):
         self.username = input("enter your username: ")
         self.password = getpass.getpass("enter your password: ")
-        i = input("To enter a group link press '1':")
-        if i is '1':
-            self.link = input("enter the link:")
-            print("taking you to group page")
-        else:
-            print("Welcome to FB homepage")
-        self.driver = webdriver.Chrome()
+        self.link = input("enter the link:")
+
+    def group_page(self):
+        self.driver.get(self.link)

```

Comment:
instead of using self.link, please pass link as an argument to the function `group_page`.

Also, give it a better name like, "open_group".